<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleRtSet</class>
 <widget class="QFrame" name="ModuleRtSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MiSans</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MiSans</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RT增稳模式</string>
   </property>
  </widget>
  <widget class="ToggleButton" name="checkBoxRTMode">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>36</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MiSans</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToggleButton</class>
   <extends>QCheckBox</extends>
   <header>togglebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
